generator/src/routes/+page.svelte

```diff
@@ -1,26 +1,27 @@
 <script lang="ts">
-  import type { Option } from "$components/FormSelectGroup/index.svelte"
+  import Tooltip from "sv-tooltip"
   import FormInputGroup from "$components/FormInputGroup/index.svelte"
   import FormSelectGroup from "$components/FormSelectGroup/index.svelte"
   import FormCheckbox from "$components/FormCheckbox/index.svelte"
   import Button from "$components/Button/index.svelte"
   import Toggle from "$components/Toggle/index.svelte"
-  import logo from "$assets/logo.svg"
-  import effect from "$assets/effect.svg"
-  import asas from "$assets/asas.svg"
-  import logoNoTag from "$assets/logo_no_tag.png"
-  import subtract from "$assets/subtract.png"
+  import logoTagsSvg from "$assets/ioxio-tags-logo.svg?url"
+  import effectSvg from "$assets/effect.svg?url"
+  import logomarkSvg from "$assets/ioxio-logomark.svg?url"
+  import logoSvg from "$assets/ioxio-logo.svg?url"
+  import QuestionSvg from "$assets/question.svg?component"
+  import subtractSvg from "$assets/subtract.png"
   import type { PageData } from "./$types"
 
   export let data: PageData
 
-  let productOption: Option
-  const onToggleProduct = (option: Option) => {
+  let productOption: string
+  const onToggleProduct = (option: string) => {
     productOption = option
   }
 
-  let signOption: Option
-  const onToggleSign = (option: Option) => {
+  let signOption: string
+  const onToggleSign = (option: string) => {
     signOption = option
   }
```

Comment:
Please use the format:
```ts
function onToggleProduct(option: Option) {
   ...
}
```
This `const name = () => {}` format creates an anonymous function which does not get a useful name in exception tracebacks and makes code harder to debug.

Reply:
Ok, but they are not used anymore.